gradle/versions.gradle

```diff
@@ -119,9 +119,9 @@ dependencyManagement {
       entry 'besu'
       entry ('core') {
         exclude group: 'com.github.jnr', name: 'jnr-unixsocket'
+        exclude group: 'org.bouncycastle', name: 'bcprov-jdk15on'
```

Comment:
Why is this still needed?

Reply:
My guess is that since the artefact name itself changed, it's not possible for gradle to automatically resolve to the jdk18 version. We're using a version of web3j that pulls in the old one. I tried upgrading web3j first and it caused other issues so aborted that, but we should revisit.

I believe the utils module is a dependency of core. In an earlier commit I had introduced the utils entry simply to be able to exclude bouncycastle. In this PR I realised I could simplify that by moving the exclude.